From: SeongJae Park <sj@kernel.org>
Date: Fri, 27 Jun 2025 09:22:19 -0700
Subject: [PATCH] mm/damon/sysfs-schemes: make damos_sysfs_action_names static

Commit 6812c66db976 ("mm/damon/sysfs-schemes: decouple from
damos_action") in mm-unstable is defining damos_sysfs_action_names as
non-static, though it is being used only inside sysfs-schemes.c file.
Make it static.

Fixes: 6812c66db976 ("mm/damon/sysfs-schemes: decouple from damos_action") # mm-unstable
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202506271655.b8yfEZIT-lkp@intel.com/

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/sysfs-schemes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/damon/sysfs-schemes.c b/mm/damon/sysfs-schemes.c
index 5243b94cec24..601360e9b521 100644
--- a/mm/damon/sysfs-schemes.c
+++ b/mm/damon/sysfs-schemes.c
@@ -1679,7 +1679,7 @@ struct damos_sysfs_action_name {
 	char *name;
 };
 
-struct damos_sysfs_action_name damos_sysfs_action_names[] = {
+static struct damos_sysfs_action_name damos_sysfs_action_names[] = {
 	{
 		.action = DAMOS_WILLNEED,
 		.name = "willneed",
-- 
2.39.5

